kie-server-parent/kie-server-wars/kie-server/src/main/assembly/assembly-servlet-container.xml

```diff
@@ -75,7 +75,6 @@
         <include>org.hibernate:hibernate-validator</include>
         <include>dom4j:dom4j</include>
         <include>xerces:xercesImpl</include>
-        <include>org.jboss.narayana.tomcat:tomcat-jta</include>
```

Comment:
@rsynek so where is it expected to have tomcat-jta on regular tomcat instances and not JWS since it is now removed from war file itself?

Reply:
@mswiderski this PR is partially outdated and - this artifact has changed meanwhile; different GAV and it's no longer uberjar. The reason why it has been removed from the assembly is that data source configuration now relies on it.

As the data source must be configured by end user, it must happen in tomcat's context.xml and thus user is required to put this artifact (and its dependencies) into tomcat's lib directory. Can you think of a more comfortable solution?

Reply:
why not include the libs that are required here? I mean since tomcat-jta was split into multiple jars why not include all these jars instead...

Reply:
The data source has a dependency on this artifact and as the DS is defined in tomcat's context.xml, it's being created before the kie-server is deployed -> CDNF.

Reply:
Before this change, the wrapping data source https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/META-INF/context.xml#L11 didn't take username and password as parameters, now it does.

Reply:
and is this the reason to externalise the entire configuration and libraries?